Source/SwiftyDropbox/Shared/Handwritten/Request+TokenRefresh.swift

```diff
@@ -0,0 +1,142 @@
+///
+/// Copyright (c) 2020 Dropbox, Inc. All rights reserved.
+///
+
+import Foundation
+import Alamofire
+
+/// Completion handler for ApiRequest.
+enum RequestCompletionHandler {
+    case dataCompletionHandler((DefaultDataResponse) -> Void)
+    case downloadFileCompletionHandler((DefaultDownloadResponse) -> Void)
+}
+
+/// Protocol defining an API request object.
+protocol ApiRequest {
```

Comment:
you should be document the protocols. The classes that implement the protocols will automatically inherit the documentation.